### PR TITLE
Use postgres with prisma instead of firebase

### PR DESCRIPTION
**Pull request by mathiasmagnusson**
Would require a little setup on the server before it would work there.

Also: I used create-t3-app, so that's where the boiler plate files that wire everything together come from.

### CODE REVIEW

**Comment by mathiasmagnusson** [comment]
Posting this here for reference later: https://developers.google.com/apps-script/reference/url-fetch/url-fetch-app#fetch(String,Object)

**Comment by ernum** [comment]
Is this PR ready for review?

**Comment by mathiasmagnusson** [comment]
> Is this PR ready for review?

Now it is!

**Comment by rikaznismi** [comment]
Nice!
Looks good now.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+
+```bash
+npx prisma migrate dev
+```
+
+Lastly, start the thing:
+
+```bash
+npm run dev
+```
```
Comment (ernum) [inline README.md:56]: I think we should create three scripts: `start.sh`, `stop.sh` and `restart.sh`. The user should be able to run these in the terminal instead to start and view the website. We don't necessarily have to do it in this PR.

File: src/locales/en.ts [hunk 2]
```diff
@@ -43,15 +43,7 @@ export default {
...
-    error: {
-      db: "Something went wrong! Try again or send an email instead to",
-      email:
-        "Your registration has been received, but we could not send a confirmation email to you.",
-      continue: "Continue anyway",
-    },
-    organizationNumberLength: "The organization number must be 10 digits",
-    organizationNumberChecksum:
-      "Invalid check digit, check that you have entered the number correctly",
+    ignoreError: "Continue anyway",
```
Comment (ernum) [inline src/locales/en.ts:46]: Why are we ignoring the error now? Is it because of trpc?

**Review by ernum — COMMENTED** [review]
So, just to confirm, we don't want to move this to production since the database is not fully set up yet, right?

File: src/locales/en.ts (hunk 2 above)
Comment (mathiasmagnusson) [reply src/locales/en.ts:46]: I have just moved the errors to its own object and renamed some keys

**Comment by mathiasmagnusson** [comment]
> So, just to confirm, we don't want to move this to production since the database is not fully set up yet, right?

Sure, but we can deploy it to `dev.ddagen.se`, then I can set it up to make sure it works

File: README.md (hunk 1 above)
Comment (ernum) [reply README.md:56]: @mathiasmagnusson Thoughts on these?

Comment (mathiasmagnusson) [reply README.md:56]: I don't think the repository should impose a specific way to run a database if it requires other dependencies (like docker), and if it doesn't there isn't much that would be in those scripts.

**Review by ernum — APPROVED** [review]

**Pull request merged**